modules/conflict_resolver/test/conflict_resolverTest.php

```diff
@@ -34,9 +36,9 @@ function testConflictResolverDoespageLoad()
     {
         $this->webDriver->get($this->url . "/conflict_resolver/");
```

Comment:
safeGet is a wrapper that @xlecours wrote around get which holds on a reference to the old DOM and waits for it to go stale, to ensure that the new page is loaded before the tests move on. It avoids race conditions where PHPUnit is trying to assert something about the new page, while the old page (in this case, the login page) is still loaded.